handler/src/main/java/io/netty/handler/ssl/ApplicationProtocolAccessor.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.handler.ssl;
+
+/**
+ * Provides a way to get the application-level protocol name from ALPN or NPN.
+ */
+interface ApplicationProtocolAccessor {
+    String getApplicationProtocol();
```

Comment:
It's a package private instance and the method is added to `SSLSession`, so I just added the `get` prefix for the consistency with `SSLSession`. I can remove it back if you really want. :-)